viewmodel/src/main/kotlin/edu/artic/viewmodel/BaseViewModelFragment.kt

```diff
@@ -3,5 +3,9 @@ package edu.artic.viewmodel
 import edu.artic.ui.BaseFragment
 
 abstract class BaseViewModelFragment<TViewModel : BaseViewModel> : BaseFragment() {
```

Comment:
your'e missing the injection stuff here too. need to port it over from activity so you can inject viewmodels in fragments tooo.